holoviews/streams.py

```diff
@@ -166,30 +166,41 @@ def trigger(cls, streams):
     def _on_trigger(self):
         """Called when a stream has been triggered"""
 
+
     @classmethod
     def _process_streams(cls, streams):
         """
         Processes a list of streams promoting Parameterized objects and
         methods to Param based streams.
         """
-        param_watch_support = util.param_version >= '1.8.0'
-        parameterizeds = [s.parameterized for s in streams if isinstance(s, Params)]
+        parameterizeds = defaultdict(list)
         valid, invalid = [], []
         for s in streams:
-            if not isinstance(s, Stream):
-                if isinstance(s, param.Parameterized) and param_watch_support:
-                    if s not in parameterizeds:
-                        s = Params(s)
-                    else:
-                        continue
-                elif util.is_param_method(s) and param_watch_support:
-                    if not hasattr(s, "_dinfo") or util.get_method_owner(s) in parameterizeds:
-                        continue
-                    else:
-                        s = ParamMethod(s)
-                else:
-                    invalid.append(s)
+            if isinstance(s, Stream):
+                pass
+            elif isinstance(s, param.Parameter):
+                s = Params(s.owner, [s.name])
+            elif isinstance(s, param.Parameterized):
+                s = Params(s)
+            elif util.is_param_method(s):
+                if not hasattr(s, "_dinfo"):
                     continue
+                s = ParamMethod(s)
+            else:
+                invalid.append(s)
+                continue
+            if isinstance(s, Params):
+                pid = id(s.parameterized)
+                if pid in parameterizeds:
+                    overlap = (set(s.parameters) & set(parameterizeds[pid]))
+                    if overlap:
+                        pname = type(s.parameterized).__name__
+                        raise ValueError('Found multiple Params streams '
+                                         'subscribing to the %s parameter(s) '
+                                         'on the %s object. Ensure that '
+                                         'you only subscribe to a parameter '
+                                         'once.' % (list(overlap), pname))
```

Comment:
What would be the effect of simply ignoring the extra subscribers to a parameter?  Would that be safe? If so, could this error simply be skipped?  If not, maybe make a note about why that wouldn't work?

Reply:
I suppose it's safe enough except for the fact that an event may get triggered multiple times. @jlstevens Any opinion here?

Reply:
By ignoring I guess I didn't just mean skipping it over, I meant to remove all but one of the subscribers for that parameter.  I guess that's not feasible?

Reply:
No you'd potentially be modifying user defined objects, which would be weird.

Reply:
Ok, sounds like a warning is best then.